go-controller/pkg/ovn/pods.go

```diff
@@ -97,11 +97,13 @@ func (oc *Controller) deleteLogicalPort(pod *kapi.Pod) {
 		klog.Errorf(err.Error())
 	}
 
-	out, stderr, err := util.RunOVNNbctl("--if-exists", "lsp-del", logicalPort)
-	if err != nil {
-		klog.Errorf("Error in deleting pod %s logical port "+
-			"stdout: %q, stderr: %q, (%v)",
-			podDesc, out, stderr, err)
+	cmd, err := oc.ovnNBClient.LSPDel(logicalPort)
+	if err == nil {
+		if err = oc.ovnNBClient.Execute(cmd); err != nil {
+			klog.Errorf("Error while deleting logical port: %s, %v", logicalPort, err)
+		}
+	} else if err != goovn.ErrorNotFound {
+		klog.Errorf(err.Error())
```

Comment:
nit: Make the log more verbose. 

`klog.Errorf("LSP for pod %s was not found in the go-ovn cache, error: %v", logicalPort, err)`